.github/workflows/test.yml

```diff
@@ -31,7 +31,9 @@ jobs:
       matrix:
         include:
             # Latest and greatest release of Nix
-          - install_url: https://nixos.org/nix/install
+          #- install_url: https://nixos.org/nix/install
+          # extra-tests/alt-store/test.sh is failing with 2.21.0
+          - install_url: https://releases.nixos.org/nix/nix-2.20.5/install
```

Comment:
Confused - is this just a workaround for something? (that security issue?)

Reply:
Yes our [`extra-test/alt-store`](https://github.com/ipetkov/crane/blob/10484f86201bb94bd61ecc5335b1496794fedb78/extra-tests/alt-store/test.sh#L8) suddenly started failing, though a new version of Nix was released as the latest version which must have some kind of regression given that the tests passed when I pinned it to the previous one

Reply:
Going to disable this test in https://github.com/ipetkov/crane/pull/556